### PR TITLE
Dtspo-3207 perftest aks initial deployment

### PR DESCRIPTION
**Pull request by avi-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/05-mis/11-jenkins.tf [hunk 1]
```diff
@@ -3,14 +3,14 @@ locals {
...
-    #   mi = "jenkins-cftptl-intsvc-mi"
-    # }
-    cftsbox = {
-      subscription = "1497c3d7-ab6d-4bb7-8a10-b51d03189ee3"
-      mi           = "jenkins-cftsbox-intsvc-mi"
+    cftptl = {
+      subscription = "5ca62022-6aa2-4cee-aaa7-e7536c8d566c"
+      mi = "jenkins-cftptl-intsvc-mi"
     }
+    #cftsbox = {
```
Comment (thomast1906) [inline components/05-mis/11-jenkins.tf:10]: Why is `cftsbox `commented?

File: environments/01-network/test.tfvars [hunk 2]
```diff
@@ -0,0 +1,38 @@
+enable_debug = "true"
+
+network_address_space                  = "10.48.64.0/18"
+aks_00_subnet_cidr_blocks              = "10.48.16.0/20"
+aks_01_subnet_cidr_blocks              = "10.48.32.0/20"
+iaas_subnet_cidr_blocks                = "10.48.1.0/24"
+application_gateway_subnet_cidr_blocks = "10.48.0.128/25"
+
```
Comment (thomast1906) [inline environments/01-network/test.tfvars:8]: The `network_address_space` does not match the proposed used subnets

File: environments/01-network/test.tfvars [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+additional_subnets = [
+]
+
+private_dns_subscription = "1497c3d7-ab6d-4bb7-8a10-b51d03189ee3"
+private_dns_zones = [
+  "perftest.platform.hmcts.net",
+  "service.core-compute-perftest.internal",
+  "service.core-compute-idam-perftest.internal",
+  "service.core-compute-perftest.internal",
+  "service.core-compute-idam-perftest.internal",
```
Comment (thomast1906) [inline environments/01-network/test.tfvars:18]: This does not reflect the current private_dns_zones in ARM
https://github.com/hmcts/cnp-aks-rbac-pipelines/blob/master/templates/vars/vnet/perftest.json#L11

File: environments/01-network/test.tfvars [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "service.core-compute-idam-sprod.internal",
+  "service.core-compute-idam-saat.internal"
+]
+
+hub = "sbox"
```
Comment (thomast1906) [inline environments/01-network/test.tfvars:23]: sbox reference? It will be the non-prod hub

File: environments/01-network/test.tfvars [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+additional_routes = [
+  {
+    name                   = "core_infra_vnet_idam_sandbox"
```
Comment (thomast1906) [inline environments/01-network/test.tfvars:27]: Update this to perftest vnet

File: environments/01-network/test.tfvars (hunk 3 above)
Comment (avi-hmcts) [reply environments/01-network/test.tfvars:18]: updated and also corrected the right subscription.

File: environments/01-network/test.tfvars (hunk 2 above)
Comment (avi-hmcts) [reply environments/01-network/test.tfvars:8]: corrected.

File: components/05-mis/11-jenkins.tf (hunk 1 above)
Comment (avi-hmcts) [reply components/05-mis/11-jenkins.tf]: corrected.

File: azure-pipelines.yml [hunk 6]
```diff
@@ -29,12 +29,12 @@ parameters:
     displayName: Environment
     type: string
     default: 'SBOX'
-    # values:
+    values:
     # - ITHC
     # - DEV
     # - DEMO
     # - SBOX
```
Comment (thomast1906) [inline azure-pipelines.yml:36]: Can you include sbox in values?

Comment (avi-hmcts) [reply azure-pipelines.yml]: done.

File: environments/01-network/test.tfvars [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+private_dns_subscription = "1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
+private_dns_zones = [
+  "perftest.platform.hmcts.net",
+  "service.core-compute-perftest.internal",
+  "service.core-compute-idam-perftest.internal"
+]
+
+hub = "nonprod"
+
+additional_routes = [
```
Comment (thomast1906) [inline environments/01-network/test.tfvars:21]: Is this the correct routetable to be updated with the two routes `core_infra_vnet_idam_perftest ``vpn`

Comment (avi-hmcts) [reply environments/01-network/test.tfvars:21]: corrected it to additional_routes_appgw.

**Review by thomast1906 — APPROVED** [review]

**Comment by thomast1906** [comment]
@avi-hmcts please resolve tf formatting issue(s) prior to merging. 
- Pipeline prechecks are failing

**Pull request merged**